bin/aws-efs-creator.ts

```diff
@@ -15,7 +15,14 @@ const app = new cdk.App();
 const appAspects = Aspects.of(app);
 
 // check environment variables
-checkEnvVariables('APP_NAME', 'OWNER', 'VPC_ID', 'CDK_DEPLOY_REGION', 'ENVIRONMENT');
+checkEnvVariables('APP_NAME',
+'OWNER',
+'VPC_ID',
+'CDK_DEPLOY_REGION',
+'ENVIRONMENT',
+'PERFORMANCE_MODE',
+'THROUGHPUT_MODE',
+);
```

Comment:
**Suggestion:** To improve readability and maintainability, consider combining the environment variable names into a single array and passing it to the `checkEnvVariables` function.
> 🚀  **High Priority** 
 > [category: enhancement, importance: 8]
```suggestion
const requiredEnvVars = [
  'APP_NAME',
  'OWNER',
  'VPC_ID',
  'CDK_DEPLOY_REGION',
  'ENVIRONMENT',
  'PERFORMANCE_MODE',
  'THROUGHPUT_MODE',
];
checkEnvVariables(...requiredEnvVars);
```